env/.env.rpc.eon.template

```diff
@@ -57,5 +57,5 @@ SCNODE_WALLET_GENESIS_SECRETS=
 SCNODE_WALLET_SEED=
 SCNODE_WALLET_MAXTX_FEE=10000000
 
```

Comment:
Let's update also here the SCNODE_NET_MAX_IN_CONNECTIONS, SCNODE_NET_MAX_OUT_CONNECTIONS and add SCNODE_FORGER_MAXCONNECTIONS=100. Double check the compose is ok about this.